etcdserver/util.go

```diff
@@ -52,3 +52,34 @@ func numConnectedSince(transport rafthttp.Transporter, since time.Time, self typ
 	}
 	return connectedNum
 }
+
+// longestConnected chooses the member with longest active-since-time.
+// It returns false, if nothing is active.
+func longestConnected(tp rafthttp.Transporter, self types.ID, membs []types.ID) (types.ID, bool) {
```

Comment:
@xiang90 @heyitsanthony I made this return `bool` to remove the `id == 0` checking.